langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -80,6 +109,13 @@ public isolated function entries(map<Type> m) returns map<[string, Type]> = @jav
 
 # Applies a function each member of a map and returns a map of the result.
 #
+# ```ballerina
+# map<float> heights = {"Carl": 1.7, "Bob": 1.778, "Max": 1.81};
+# map<float> heightsCm = heights.map(function(float h) returns float {
```

Comment:
```suggestion
# map<float> heightsCm = heights.map(isolated function(float h) returns float {
```

Reply:
Why do we need to tag this function as isolated? Is is mandatory to have an isolated function?